layers/+tools/transmission/packages.el

```diff
@@ -29,50 +29,94 @@
     :defer t
     :init (progn
             (spacemacs/set-leader-keys "att" 'transmission)
+
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ma" "add")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mg" "goto")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ms" "set")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mt" "toggle")
+            (spacemacs/declare-prefix-for-mode 'transmission-mode "mQ" "Queue")
             (spacemacs/set-leader-keys-for-major-mode 'transmission-mode
-              "gt" 'transmission-files
-              "at" 'transmission-add
-              "sd" 'transmission-set-download
-              "gp" 'transmission-peers
-              "gi" 'transmission-info
-              "sl" 'transmission-set-ratio
+              "D"  'transmission-delete
               "m"  'transmission-move
-              "r"  'transmission-remove
-              "ts" 'transmission-toggle
+              "q"  'transmission-quit
+              "R"  'transmission-remove
+              "v"  'transmission-verify
+              "S"  'tabulated-list-sort
+
+              ;; add
               "aa" 'transmission-trackers-add
+              "at" 'transmission-add
+
+              ;; goto
+              "gi" 'transmission-info
+              "gp" 'transmission-peers
+              "gt" 'transmission-files
+
+              ;; set
+              "sb" 'transmission-set-bandwidth-priority
```

Comment:
should this be "sp" as for set priority?
bandwidth cannot be set.

Reply:
This was also brought over as it was from before. I failed to spot that it does not match with the binding in `transmission-files-mode` where it is bound to `sp`. This should be changed.